action.yaml

```diff
@@ -17,7 +17,7 @@ runs:
           echo "unit_test=x" >> $RUNNER_TEMP/icons.properties
 
 
-          if [ "${{ github.event_name }}" == "push" ]; then
+          if [ "${{ github.event_name }}" == "push" ] || [ "${{ github.event_name }}" == "workflow_dispatch" ]; then
```

Comment:
If instead of this, what would be the impact of having `if [ "${{ github.event_name }}" != "pull_request" ]; then`? Would it be better or worse... what do you think?